src/main/java/dev/dirs/Util.java

```diff
@@ -183,27 +194,33 @@ private static void initBase64Encoding() {
     }
   }
 
-  private static String[] runCommands(int expectedResultLines, Charset charset, String... commands) {
-    final ProcessBuilder processBuilder = new ProcessBuilder(commands);
-    Process process;
-    try {
-      process = processBuilder.start();
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
+  private static Process makeProcessForWindows(String ...commands) throws IOException {
+    final String[] fullCommand = new String[commands.length + 2];
+    fullCommand[0] = "cmd.exe";
+    fullCommand[1] = "/C";
+    for (int i = 0; i < commands.length; ++i) fullCommand[i + 2] = commands[i];
+    final ProcessBuilder processBuilder = new ProcessBuilder(fullCommand);
+    final Map<String, String> env = processBuilder.environment();
+    final String path = env.get("Path");
+    final String toAppend = "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\;C:\\Program Files\\PowerShell\\7";
+    env.put("Path", (path == null ? "" : path) + toAppend);
```

Comment:
Isn't a `;` necessary between `path` and `toAppend` here?

Reply:
Yes. That was an oversight. It should be `File.pathSeparator`.

Reply:
https://github.com/dirs-dev/directories-jvm/compare/1cac56cc0f36f74deead7669338af598ac645f3f..149929c21fbfb4fa5d541ac98c190b71b13aea24